app/components/search/result-cards/document.hbs

```diff
@@ -1,27 +1,38 @@
-<AuCard @shadow={{true}} @size="small" as |c|>
+<AuCard @shadow={{true}} @size="small" class="result-card" as |c|>
   <c.header>
     <div class="au-u-flex au-u-flex--vertical-center au-u-flex--spaced-tiny">
       <p class="au-u-para-small au-u-muted au-u-medium">
-        {{t "created"}}
         {{date @created}}
       </p>
     </div>
-    <AuHeading @level="3" @skin="4">
+    <AuHeading @level="3" @skin="6">
       <AuLink
         class="au-u-padding-none"
-        @route="agenda.agendaitems.agendaitem"
+        @route="document"
         @model={{@documentId}}
+        @icon="document"
+        @iconAlignment="left"
       >
-        {{@title}}
+        {{@fileName}}
```

Comment:
Pretty sure this needs a <SanitizeHtml> around it. Will fix on dev with any other bugs so we can merge now